packages/ipfs-core/src/components/bitswap/stat.js

```diff
@@ -30,7 +30,7 @@ module.exports = ({ bitswap }) => {
   /**
    * @type {Stat<{}>}
    */
-  async function stat (options) { // eslint-disable-line require-await, @typescript-eslint/no-unused-vars
+  async function stat (_options) { // eslint-disable-line require-await
```

Comment:
Why the underscore?

Reply:
TS will complain about unused variables (that's why we had a comment before saying `@typescript-eslint/no-unused-vars`, but it also recognized `_` prefixed bindings as things that may not be used and won't complain.

I like later pattern because it becomes I find it more helpful while reading code, but I could also either:

1. Remove the argument
2. Leave the comment to tell TS to ignore this.

Reply:
Also, if we start using these jsdoc comments to generate a doc website, won't the argument name show up as `_options`?  Definitely don't want that.

Reply:
Can I do `void options` in the source ? That would make TS happy and it's going to be more clear that just `@ts-ignore`

Reply:
I'll do comments